src/cpp/rtps/builtin/discovery/database/DiscoveryDataBase.hpp

```diff
@@ -540,7 +540,7 @@ class DiscoveryDataBase
     std::atomic<bool> enabled_;
 
     // Whether it has been a new entity discovered or updated in this subroutine loop
-    std::atomic<int> entities_updated_;
+    std::atomic<int> new_entities_;
```

Comment:
I think this name could be misunderstood, because this value does not count new entities but checks whether the PDPServer must add DATA(p)s to History, and that occurs with new entities **AND** with update or disposal of old ones.

Maybe we could call it `ddb_updates_` or `new_updates_`